typing/env.ml

```diff
@@ -1616,6 +1635,11 @@ and components_of_module_maker (env, sub, path, mty) =
             let decl' = Subst.type_declaration sub decl in
             let constructors =
               List.map snd (Datarepr.constructors_of_type path decl') in
+            let ext_constructors =
```

Comment:
I was the one out of mind, `ext_constructors` is indeed always empty and thus useless here, fixed in
commit 04c12971d2c295fb40